### PR TITLE
Added LinearSearch,QuickSort,InsertionSort

### PR DESCRIPTION
**Pull request by shubham2704**
## Algorithms added:
QuickSort
InsertionSort
LinearSearch

## Language:
python

### CODE REVIEW

**Comment by rohan8594** [comment]
@shubham2704 I'll review and merge this over the weekend. Can you pls in the meantime create a separate issue mentioning these algorithms and reference that issue on here. So that no one adds these again.

**Comment by shubham2704** [comment]
> @shubham2704 I'll review and merge this over the weekend. Can you pls in the meantime create a separate issue mentioning these algorithms and reference that issue on here. So that no one adds these again.

yeah sure i will do it

File: algorithms/Sorting/InsertionSort.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+def insertionSort(arr):
+   for i in range(1, len(arr)):
+      key = arr[i]
+      # Move elements of arr[0..i-1], that are greater than key,
+      # to one position ahead of their current position
+      j = i-1
+      while j >=0 and key < arr[j] :
+         arr[j+1] = arr[j]
+         j -= 1          
+       arr[j+1] = key
```
Comment (rohan8594) [inline algorithms/Sorting/InsertionSort.py]: Please fix the indentation here. Python will throw an indentation error

**Review by rohan8594 — CHANGES_REQUESTED** [review]

**Comment by shubham2704** [comment]
@rohan8594 fixed the indentation .☑

**Review by rohan8594 — APPROVED** [review]

**Comment by rohan8594** [comment]
@all-contributors please add @shubham2704 for code

**Pull request merged**